### DISCUSSION TITLE
Types imports as `any` in VUE components.

### DISCUSSION THREAD

**Discussion by maxorshak** [trpc/trpc#3514] [Q&A]
Hello, does anybody know what the reason of unexpected types behavior on client using VUE? 
Vue component recognizes imported trpc types as any. Here is an example below.
Client creation:
<img width="684" alt="Screenshot 2022-12-28 at 18 47 10" src="https://user-images.githubusercontent.com/71508924/209846047-26cbd2ed-5b14-4960-861a-d11193886679.png">
<img width="763" alt="Screenshot 2022-12-28 at 21 31 51" src="https://user-images.githubusercontent.com/71508924/209863190-19295cf5-a2e4-4278-8244-00c43e970a3c.png">

Type usage in a VUE file:
<img width="512" alt="Screenshot 2022-12-28 at 18 47 33" src="https://user-images.githubusercontent.com/71508924/209846053-6034d477-1dc9-41d4-8be1-535fbf95821d.png">
<img width="516" alt="Screenshot 2022-12-28 at 18 47 38" src="https://user-images.githubusercontent.com/71508924/209846057-2fa2ff16-adc7-420e-8dcc-7c9711c7a503.png">

**Comment by moritzruth** [comment]
Did you solve this problem? I’m experiencing the same.

**Reply by maxorshak** [reply]
Yes. Make sure that the trpc server and client versions on both backend/front apps match.
I use these versions:
```
"@trpc/client": "^10.9.0",
"@trpc/server": "^10.9.0",
```
Also you have to run `build` on the backend.
Also restart typescript server in the command palette in VS Code.
